docs/examples/Customized/CustomizedContent.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+  Customized,
+  Cross,
+} from 'recharts';
+
+const data = [
+  {
+    name: 'Page A',
+    uv: 4000,
+    pv: 2400,
+    amt: 2400,
+  },
+  {
+    name: 'Page B',
+    uv: 3000,
+    pv: 1398,
+    amt: 2210,
+  },
+  {
+    name: 'Page C',
+    uv: 2000,
+    pv: 9800,
+    amt: 2290,
+  },
+  {
+    name: 'Page D',
+    uv: 2780,
+    pv: 3908,
+    amt: 2000,
+  },
+  {
+    name: 'Page E',
+    uv: 1890,
+    pv: 4800,
+    amt: 2181,
+  },
+  {
+    name: 'Page F',
+    uv: 2390,
+    pv: 3800,
+    amt: 2500,
+  },
+  {
+    name: 'Page G',
+    uv: 3490,
+    pv: 4300,
+    amt: 2100,
+  },
+];
+
+// using Customized gives you access to all relevant chart props
+const CustomizedCross = (props) => {
+  const { width, height, stroke, fill, formattedGraphicalItems } = props;
+  console.log(props);
+  // access datapoints formatted for the chart
+  console.log(formattedGraphicalItems);
+  // get first series in chart
```

Comment:
Should this be in the example? If yes, logging props should be sufficient. FormattedGraphicalItems would be logged twice.

Reply:
I logged so the user can see what all gets passed to Cystomized as props since it isn't documented yet. I logged again to explicitly show what was in the graphical items (because things are a few levels deep). 

Going to merge this (from mobile) and I'll do a small PR to clean up the extra log and maybe add a better description to customized